WordPress/src/main/java/org/wordpress/android/viewmodel/posts/PostListViewModel.kt

```diff
@@ -833,6 +810,41 @@ class PostListViewModel @Inject constructor(
         _snackbarAction.postValue(snackbarHolder)
     }
 
+    fun scrollToPost(localPostId: Int) {
+        val data = pagedListData.value
+        if (data != null) {
+            _scrollToPosition.value = findItemListPosition(data, localPostId) ?: run {
```

Comment:
Should we set the `_scrollToPosition` only if the position is found instead of setting it to `null` when we can't find it? I feel like we can't guarantee that the observer will handle the `null` value correctly especially since we are using a non-null value for the `LiveData`.

Reply:
👍